ssl/ssl_lib.cc

```diff
@@ -2760,6 +2760,10 @@ void SSL_CTX_set_tmp_dh_callback(SSL_CTX *ctx,
 void SSL_set_tmp_dh_callback(SSL *ssl, DH *(*cb)(SSL *ssl, int is_export,
                                                  int keylength)) {}
 
+long SSL_CTX_set_dh_auto(SSL_CTX *ctx, int onoff) {
+    return 0;
```

Comment:
Is this always safe? Could customers accidentally think they're turning on FFDH even though AWS-LC doesn't support it at all? Should we require onoff to always be 0?

Reply:
https://www.openssl.org/docs/man3.1/man3/SSL_CTX_set_dh_auto.html
From above link:
"All of these functions/macros return 1 for success or 0 on error."

So returning 0 should always be treated as an error, I can update documentation with this info.